Automattic/wp-calypso
# Bulk Domain Transfer: Apply new routing

Rename `bulk-domain-transfer` to `domain-transfer`
Making the flow available on `setup/domain-transfer/domains` instead of `setup/bulk-domain-transfer/domains`


## Proposed Changes

* Rename `bulk-domain-transfer` to `domain-transfer`

## Testing Instructions



1. Live link and visit ` /setup/domain-transfer`
2. Check that the flow is reachable and working well
